### DISCUSSION TITLE
How to install web3-os instead of ubuntu?

### DISCUSSION THREAD

**Discussion by ghost** [web3os-org/kernel#3] [Q&A] [answered]
Hi.

I would like to know how can I install web3os on my computer, so... how to generate iso file.

- How to install web3-os instead of ubuntu?
- Is there any file in iso?
- how can I install web3os on my computer?
- How to generate iso file from web3-os?

**Comment by mathiscode** [comment, marked as answer]
@codehangen Excellent question! Actually web3os is completely web-based, so you don't install it locally.

However, you can install it as a Progressive Web App (PWA) which is supported on most browsers and operating systems. Just run `install` from the [web3os](https://web3os.sh) command line.

This allows you to run it from anywhere on any device, and I'm working on file syncing so you can connect dropbox, google drive, ipfs, etc to have your files with you wherever you go as well.

Currently, it's main demographic will be Javascript/WASM developers - there are many apps already included, but it's going to take a lot of devs to develop more apps for it and make it a decent ecosystem.

To use web3os just go to [https://web3os.sh](https://web3os.sh) in any browser (Chrome has better support for experimental browser APIs).

Also check out the docs at [https://docs.web3os.sh](https://docs.web3os.sh)

**Reply by ghost** [reply]
Hi mathiscode, thank you for feedback.
